src/test-utils/mock-data.ts

```diff
@@ -300,6 +300,30 @@ export const MockPerformanceTiming: PerformanceTiming = {
     toJSON: () => ({})
 };
 
+export const mockPerformanceEntry = (partialEntry?: any): PerformanceEntry => ({
+    name: 'localhost',
+    entryType: 'resource',
+    duration: 10,
+    startTime: 0,
+    ...partialEntry
+});
+
+export const mockPerformanceResourceTiming = (
+    partialEntry?: any
+): PerformanceResourceTiming => ({
+    ...mockPerformanceEntry(),
+    ...partialEntry,
+    entryType: 'resource'
+});
+
+export const mockPerformanceNavigationTiming = (
+    partialEntry?: any
+): PerformanceNavigationTiming => ({
+    ...mockPerformanceEntry(),
+    ...partialEntry,
+    entryType: 'navigation'
+});
+
```

Comment:
nit: We can omit these. The tests for performance-utils are already minimal.

Reply:
deleted